frontend-react/src/components/MessageTracker/MessageReceivers.tsx

```diff
@@ -74,6 +74,8 @@ const StatusEnum = {
     READY: "READY",
 };
 
+const noDataString = "N/A";
```

Comment:
MINOR POINT, but mind updating the variable name here to be something like `NO_DATA_STRING` instead?  Just signifying that it's a static value for the rest of the app lifetime